src/vs/workbench/browser/parts/titlebar/menubarControl.ts

```diff
@@ -133,7 +131,7 @@ export abstract class MenubarControl extends Disposable {
 		this.menuUpdater.schedule();
 	}
 
-	protected calculateActionLabel(action: IAction | IMenubarMenuItemAction): string {
+	protected calculateActionLabel(action: { id: string; label: string; }): string {
```

Comment:
@bpasero this was the one to which I was referring but I see it is the only one so no worries there